desktop/core/src/desktop/views.py

```diff
@@ -61,20 +56,26 @@
 from desktop.lib.paths import get_desktop_root
 from desktop.lib.thread_util import dump_traceback
 from desktop.lib.view_util import is_ajax
-from desktop.log.access import access_log_level, access_warn, AccessInfo
-from desktop.log import set_all_debug as _set_all_debug, reset_all_debug as _reset_all_debug, \
-                        get_all_debug as _get_all_debug, DEFAULT_LOG_DIR
-from desktop.models import Settings, hue_version, _get_apps, UserPreferences
+from desktop.log import (
+  DEFAULT_LOG_DIR,
+  get_all_debug as _get_all_debug,
+  reset_all_debug as _reset_all_debug,
+  set_all_debug as _set_all_debug,
+)
+from desktop.log.access import AccessInfo, access_log_level, access_warn
+from desktop.models import Settings, UserPreferences, _get_apps, hue_version
 from libsaml.conf import REQUIRED_GROUPS, REQUIRED_GROUPS_ATTRIBUTE
-from useradmin.models import get_profile
-from useradmin.models import User
+from useradmin.models import User, get_profile
+
+standard_library.install_aliases()
```

Comment:
Good to remove

Reply:
Removed line 70: standard_library.install_aliases()